src/odemis/acq/stream/_base.py

```diff
@@ -1050,3 +1050,11 @@ def getBoundingBox(self):
         c = md[model.MD_POS]
         w, h = shape[1] * pxs[0], shape[0] * pxs[1]
         return (c[0] - w / 2, c[1] - h / 2, c[0] + w / 2, c[1] + h / 2)
+
+    def getRawMetadata(self):
+        """
+        Gets the raw metadata structure from the stream.
+        """
+        if isinstance(self.raw[0], model.DataArray):
+            return self.raw[0].metadata
```

Comment:
I think it should return a tuple or list of dict, corresponding to the metadata of each DataArray in .raw, instead of returning just the first one.
Also, it needs to support tiled data (cf `._das`, and gui.cont.streams).

Ideally, you should also update the code that is already doing tricks to get the raw data. There is at least some code in gui.cont.streams, and probably also in the gui.util.img.